### PR TITLE
Adds Flax BERT finetuning example on GLUE

### PR DESCRIPTION
**Pull request by marcvanzee**
# What does this PR do?

Adds Flax BERT finetuning example which finetunes on one of the GLUE tasks.

I evaluated all tasks 5 times and added the average runs, the best run. and stdev in a table in the README. I used the seed of the best run as the default.

I also ran all experiments on three devices: 8 Cloud TPU-v3, 1 Cloud TPU-v3, 1 P100 GPU. I compared the runtimes and put them in another table in the README.

This PR was discussed over Slack with @patrickvonplaten and @sgugger .

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [X] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/flax/text-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+```bash
+export TASK_NAME=mrpc
+
+python run_flax_glue.py \
+  --model_name_or_path bert-base-cased \
+  --task_name $TASK_NAME \
+  --max_length 128 \
+  --learning_rate 2e-5 \
+  --num_train_epochs 3 \
+	--per_device_train_batch_size 4 \
```
Comment (patrickvonplaten) [inline examples/flax/text-classification/README.md:37]:
```suggestion
  --per_device_train_batch_size 4 \
```

**Comment by patrickvonplaten** [comment]
Changes were done according to discussion [here](https://github.com/huggingface/transformers/pull/11593)
